### PR TITLE
<Image/> - showBorder prop 

### PR DESCRIPTION
**Pull request by roibWix**
### 🔦 Summary

...

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6379

File: src/Image/Image.st.css [hunk 1]
```diff
@@ -20,3 +25,7 @@
   object-position: -100vw;
   object-fit: none;
 }
+
+.root:showBorder {
+  border: 1px solid calc_rgba(value(D10), 0.12);
```
Comment (EgoziE) [inline src/Image/Image.st.css:30]: @milkyfruit We don't have `D10-12` currently
Do you want us to add it to the palette or use a color we already have?

File: src/Image/docs/examples/showBorder.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/* eslint-disable */
+<Layout cols={1}>
+  <Image src="example.jpg" showBorder={true}/>
+</Layout>
```
Comment (EgoziE) [inline src/Image/docs/examples/showBorder.js:4]: Merge all examples into one file

Comment (EgoziE) [reply src/Image/docs/examples/showBorder.js:4]: remove ={true}

File: src/Image/test/Image.visual.js [hunk 3]
```diff
@@ -164,6 +164,25 @@ const tests = [
...
+      {
+        it: 'should not show border if no showBorder prop passed',
+        props: {},
+      },
+      {
+        it: 'should not show border if showBorder prop is false',
+        props: {
+          showBorder: false,
+        },
+      },
```
Comment (EgoziE) [inline src/Image/test/Image.visual.js:179]: remove

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/Image/docs/examples/showBorder.js (hunk 2 above)
Comment (roibWix) [reply src/Image/docs/examples/showBorder.js:4]: done

File: src/Image/test/Image.visual.js (hunk 3 above)
Comment (roibWix) [reply src/Image/test/Image.visual.js:179]: done

**Review by milkyfruit — CHANGES_REQUESTED** [review]
@roibWix great work! I have 2 small requests:

- [ ] `showBorder` prop makes image 2px larger. It should remain the same size
- [ ] Example should have a description:
"When content is light, the image can appear with a border to make image ratio visible."

File: src/Image/Image.st.css (hunk 1 above)
Comment (milkyfruit) [reply src/Image/Image.st.css:30]: yes, let's add it to the palette.

Comment (roibWix) [reply src/Image/Image.st.css:30]: done

**Review by EgoziE — APPROVED** [review]

**Pull request merged**